synapse/handlers/sync.py

```diff
@@ -153,6 +157,7 @@ class ArchivedSyncResult:
     timeline: TimelineBatch
     state: StateMap[EventBase]
     account_data: List[JsonDict]
+    preview: Optional[JsonDict]
```

Comment:
This was only added because MyPy complained. It's not ever used in the code. See below.

Reply:
This is causing some test flakiness, should probably just append this field i.e. UnreadCount